packages/sources/bitex/src/endpoint/tickers.ts

```diff
@@ -1,7 +1,8 @@
 import { Requester, Validator } from '@chainlink/ea-bootstrap'
 import { ExecuteWithConfig, Config } from '@chainlink/types'
 
-export const NAME = 'tickers'
+export const supportedEndpoints = ['tickers']
+const NAME = 'tickers'
```

Comment:
I would take this line out and just hard code line 21 with `tickers` in it.

Reply:
Ok I will look for instances of this issue and fix